app/ts/AddressBook.tsx

```diff
@@ -60,9 +60,9 @@ export function ConfirmaddressBookEntryToBeRemoved(param: ConfirmaddressBookEntr
 						<img src = '../img/address-book.svg'/>
 					</span>
 				</div>
-				<p class = 'card-header-title'>
+				<div class = 'card-header-title'>
 					<p className = 'paragraph'> { `Remove ${ ActiveFilterSingle[param.category] }` } </p>
-				</p>
+				</div>
```

Comment:
This could be a single h / p / span element.

Reply:
hmm, I like that the card header consists of card header parts, which then contain something in them. Instead of "directly writing" to header

Reply:
Is the idea that you may add more complexity to the header in the future?